frameworks/mlr3automl/exec.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import os
+
+from amlb.benchmark import TaskConfig
+from amlb.data import Dataset
+from amlb.utils import dir_of, run_cmd, Timer
+
+log = logging.getLogger(__name__)
+
+
+def run(dataset: Dataset, config: TaskConfig):
+    #TODO: use rpy2 instead? not necessary here though as the call is very simple
+    log.info("\n**** mlr3automl (R) ****\n")
```

Comment:
new requirement, you need to find a way to retrieve the exact version of your package here.
```
    log.info(f"\n**** mlr3automl (R) [{version}] ****\n")
    save_metadata(config, version=version)
```
If this is not possible, then use the following:
```
    log.info(f"\n**** mlr3automl (R) [{config.framework_version}] ****\n")
    save_metadata(config)
```